recipes/qt5/build.sh

```diff
@@ -1,11 +1,16 @@
 #!/bin/bash
 
-# Main variables
-# --------------
-BIN=$PREFIX/lib/qt5/bin
-QTCONF=$BIN/qt.conf
-
+# Download QtWebkit
+curl "http://linorg.usp.br/Qt/community_releases/5.6/${PKG_VERSION}/qtwebkit-opensource-src-${PKG_VERSION}.tar.xz" > qtwebkit.tar.xz
+unxz qtwebkit.tar.xz
+tar xf qtwebkit.tar
+mv qtwebkit-opensource-src* qtwebkit
+patch -p0 < ${RECIPE_DIR}/0001-qtwebkit-old-ld-compat.patch
+patch -p0 < ${RECIPE_DIR}/0002-qtwebkit-ruby-1.8.patch
+patch -p0 < ${RECIPE_DIR}/0003-qtwebkit-O_CLOEXEC-workaround.patch
+patch -p0 < ${RECIPE_DIR}/0004-qtwebkit-CentOS5-Fix-fucomip-compat-with-gas-2.17.50.patch
 
+rm qtwebkit.tar
```

Comment:
@mingwandroid, this whole WebKit block needs to be executed only on Linux. We're going to provide WebEngine in OS X.